server/apps/distributed-app/docs/modules/ROOT/pages/operate/guide.adoc

```diff
@@ -137,7 +137,7 @@ Currently, an administrator can monitor indexation failures through
 `ERROR` log review. You can as well
 xref:operate/webadmin.adoc#_listing_failed_events[list failed events] by
 looking with the group called
-`org.apache.james.mailbox.elasticsearch.v7.events.ElasticSearchListeningMessageSearchIndex$ElasticSearchListeningMessageSearchIndexGroup`.
+`org.apache.james.mailbox.elasticsearch.v8.events.ElasticSearchListeningMessageSearchIndex$ElasticSearchListeningMessageSearchIndexGroup`.
```

Comment:
We may need to add manual unbindings in RabbitMQ to avoid issues in upgrade instructions.

Reply:
> Will this cause issue?

like every time we change a group listener in dead letter... that design based on the FQDN isn't good IMO, it just breaks if you replay dead letters and you still had events in there with the old one after an upgrade.

> We may need to add manual unbindings in RabbitMQ to avoid issues in upgrade instructions.

You mean like replay your dead letters, make sure it's empty of events of that group before upgrading James?

Reply:
I mean manually deletes the bindings from the event queues in RabbitMQ